tox.ini

```diff
@@ -50,7 +50,6 @@ commands =
 description = Run unit tests
 commands =
     poetry install --with unit
-    poetry export -f requirements.txt -o requirements.txt
```

Comment:
Renovate will update the lock and requirements, so we don't really need to export everywhere.